From f1f3f597d7adbea03ade4cd469631dc828b48f60 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 27 Jun 2020 09:54:09 +0200
Subject: [PATCH 1/1] crypto/fsl: correct printf() statement.

The sequence of arguments should match the format string.
For printing unsigned numbers we should use %u.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 drivers/crypto/fsl/jobdesc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/crypto/fsl/jobdesc.c b/drivers/crypto/fsl/jobdesc.c
index decde64078..fbc1aeddee 100644
--- a/drivers/crypto/fsl/jobdesc.c
+++ b/drivers/crypto/fsl/jobdesc.c
@@ -102,8 +102,8 @@ int caam_page_alloc(uint8_t page_num, uint8_t partition_num)
 
 	/* if the page is not owned => problem */
 	if ((temp_reg & SMCSJR_PO) != PAGE_OWNED) {
-		printf("Allocation of page %d in partition %d failed 0x%X\n",
-		       temp_reg, page_num, partition_num);
+		printf("Allocation of page %u in partition %u failed 0x%X\n",
+		       page_num, partition_num, temp_reg);
 
 		return ERROR_IN_PAGE_ALLOC;
 	}
-- 
2.20.1

